src/engines/kotor/module.h

```diff
@@ -194,6 +201,10 @@ class Module : public KotOR::Object, public KotOR::ObjectContainer {
 
 	Common::ScopedPtr<IngameGUI> _ingame; ///< The ingame ui.
 
+	// .--- Global values
+	std::map<Common::UString, int> _globalNumbers;
```

Comment:
This map needs to be cleared when we leave the game (to go back to the main menu or load a saved game), i.e. in `Module::unload()` with completeUnload == true.